### ISSUE TITLE
[FR] Autoload Filament - No Runout Before Insert

### ISSUE THREAD

**Issue by ETE-Design** [MarlinFirmware/Marlin#23667]
### Describe the feature you want

Short Version, would like to get this code working in Marlin:

https://github.com/niccoreyes/Marlin/commit/73af75bf3359ce7236410a1aefd0871c56e5b5b4

Long Version:
Would be great to have a feature like:
"#define FILAMENT_CHANGE_RESUME_ON_INSERT"

But as an extra option have something like "First Time Preheat", so if none of the Filament Runout Sensor has been active then, when filament is inserted:

- Load X amount of Filament
- Preheat
- Purge

Just like the feature Prusa already use...

**Comment by sarvenn** [comment]
Nice request. I thought prusa mk3 had a similar feature.

**Comment by ETE-Design** [comment]
@sarvenn They have :-) Just need someone with the knowlage to transform it to Marlin 2.0

**Comment by sarvenn** [comment]
After insertion there should be a parameter to adjust the delay time for seqeunce to start.

Also it could be suitable for bowden printers as well. And with the delay function, it could be suitable for printers which had their runout sensor at a distance to extruder.

**Comment by ETE-Design** [comment]
@sarvenn Agreed, but first we need to find a person who is willing to make the code, or try to merge the code I linked to...

**Comment by sarvenn** [comment]
I hope a developer will take care of this feature addition request.
